### DISCUSSION TITLE
Provenance des ratings

### DISCUSSION THREAD

**Discussion by mifrey00** [Kodi-vStream/venom-xbmc-addons#3984] [Support & aide] [answered]
Bonjour à tous. Je ne parviens pas à trouver la provenance des ratings des films. Par exemple le film Old People https://www.themoviedb.org/movie/999722-old-people me donne dans VStream un rating de 6.4 et 58 votes. Ca ne correspond a aucun des sites que je vois : 
- TMDB : Rating 5.8, 168 votes (https://api.themoviedb.org/3/movie/999722?api_key=xxx)
- IMDB : Rating 5.1, 4300 votes (https://www.imdb.com/title/tt21933110/)

De manière générale le nombre de votes est très faible pour les films. D'où provient le rating et le nombre de votes, et y a-t-il moyen d'utiliser IMDB par exemple ?

**Comment by sizanic** [comment, marked as answer]
Bonjour,

si, il s'agit bien des notations TMDB.

Le souci vient du cache de vStream qui va sauvegarder la note qui correspond à la première interrogation des informations sur le site TMDB.

Il suffit de vider le cache de vStream, pour avoir la dernière note (jusqu'au prochain forçage).

![image](https://user-images.githubusercontent.com/47307622/209717267-c392e60a-48d5-4009-a9ca-99872d936969.png)


![image](https://user-images.githubusercontent.com/47307622/209717050-2740925c-aee2-46c3-832b-e2fa8504c4ee.png)

**Reply by mifrey00** [reply]
Super, c'est clair, merci !  Tiens, y a-t-il moyen de noter un film depuis VStream ou Kodi ?

**Comment by sizanic** [comment]
Les options TMDB proposées par vStream : 
![image](https://user-images.githubusercontent.com/47307622/209725218-de337a91-97b6-4b14-87e2-b896dc3058f3.png)

**Reply by mifrey00** [reply]
Je fais comment pour faire apparaître ce menu ? Je n'ai pas ca dans le menu contextuel :
![screenshot00000](https://user-images.githubusercontent.com/10077480/209848194-a17247fd-886f-4d63-a4fd-3959641ab008.png)

**Comment by sizanic** [comment]
Je vais regarder pour que lorsqu'on vote, cela provoque le rafraichissement des metadonnées (ce qui permettrait de voir son vote pris en compte).

**Reply by sizanic** [reply]
Annonce vraiment hors sujet mais le service proposé vaut le coup d'oeil 😂
Augmenter la note IMDb de son contenu en supprimant les mauvaises notations, et en mettant des bonnes ! 😄
